packages/vite/package.json

```diff
@@ -1,18 +1,36 @@
 {
 	"name": "@prefresh/vite",
 	"version": "0.8.1",
-	"module": "src/index.js",
-	"main": "dist/src/index.js",
+	"module": "dist/index.mjs",
+	"main": "dist/index.js",
 	"exports": {
 		".": {
-			"import": "./src/index.js",
-			"require": "./dist/src/index.js"
+			"import": "./dist/index.mjs",
+			"require": "./dist/index.js"
+		},
+		"./runtime": {
+			"import": "./runtime/index.mjs",
+			"require": "./runtime/index.js"
+		},
+		"./utils": {
+			"import": "./utils/index.mjs",
+			"require": "./utils/index.js"
 		},
 		"./package.json": "./package.json",
 		"./": "./"
 	},
+	"modes": {
```

Comment:
This is for `bundt` – without it it'd default to `src/index.js` only, but _with it_ it produces multiple targets:

```
modes: {
  "<target directory>": "<entry file>"
}
```